src/core/solvers/ddp.cpp

```diff
@@ -456,7 +457,7 @@ const std::vector<Eigen::VectorXd>& SolverDDP::get_Qx() const { return Qx_; }
 
 const std::vector<Eigen::VectorXd>& SolverDDP::get_Qu() const { return Qu_; }
 
-const std::vector<Eigen::MatrixXd>& SolverDDP::get_K() const { return K_; }
+const std::vector<MatrixXdRowMajor>& SolverDDP::get_K() const { return K_; }
```

Comment:
Could you verify that this is properly bind by EigenPy? (@jcarpent -- you might comment here)

Reply:
Should work since Feb 2020, cf. https://github.com/stack-of-tasks/eigenpy/blob/b55c10dde71f81aaa51e199f4b64071d31e882eb/src/matrix-double.cpp#L12

Reply:
Thanks for the answer :)